analyzer/src/main/java/org/jboss/gm/analyzer/alignment/DAAlignmentService.java

```diff
@@ -46,28 +46,40 @@ public DAAlignmentService(Configuration configuration) {
 
     @Override
     public Response align(AlignmentService.Request request) throws RestException {
-        final List<ProjectVersionRef> translateRequest = new ArrayList<>(request.getDependencies().size() + 1);
-
         if (dependencySource == NONE) {
             logger.warn("No dependencySource configured ; unable to call endpoint");
             return new Response(Collections.emptyMap());
         }
 
-        translateRequest.addAll(request.getProject());
-        translateRequest.addAll(request.getDependencies());
+        // XXX: lookupVersions expects List, but AlignmentService.Request.getDependencies() returns Set
```

Comment:
This is not a big deal, but I put a comment here just in case we decide to change it later.

Reply:
+1, it might be simpler to change `AlignmentService.Request`